src/MusicStore/Platform.cs

```diff
@@ -82,5 +82,14 @@ public bool IsRunningOnNanoServer
                 return _isNano.Value;
             }
         }
+
+        //Sql client not available on mono, non-windows, or nano
+        public bool UseInMemoryStore
```

Comment:
Seems ok to me. Maybe @kichalla has suggestions since he's pretty familiar with this piece of code.